src/components/users/userProfile/UserProfile.js

```diff
@@ -15,26 +16,24 @@ const UserProfile = () => {
   const { id } = useParams();
 
   useEffect(() => {
-      handleFindProfile()
-  })
-
-  const handleFindProfile = () => {
-    client.get(`/user/${id}`)
-    .then(res => {
-        setProfile(res.data.data.user)
-    })
-    .catch(err => console.log(err.response))
-  };
+      client
+        .get(`/user/${id}`)
+        .then((res) => {
+          setProfile(res.data.data.user);
+        })
+        .catch((err) => console.log(err.response));
+  }, []);
```

Comment:
The cypress tests are failing due to this warning:

>  Line 25:6:  React Hook useEffect has a missing dependency: 'id'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

I think you'll need to resolve that warning before the build passes and you can merge.